Funcky/Internal/ComparisonResult.cs

```diff
@@ -0,0 +1,9 @@
+namespace Funcky.Internal
+{
+    internal static class ComparisonResult
```

Comment:
Where is this used?

Reply:
oh it wasn't when you reviewed, but is now used again :)
https://github.com/polyadic/funcky/blob/674e5da2af01fab276dc3c182be979b1d60dd596/Funcky/Monads/Option/OptionComparer.cs#L49-L53